finality-aleph/src/new_network/component.rs

```diff
@@ -0,0 +1,42 @@
+use crate::new_network::{DataNetwork, SendError};
+use aleph_bft::Recipient;
+use codec::Codec;
+use futures::channel::mpsc;
+use std::sync::Arc;
+use tokio::sync::Mutex;
+
+/// For sending arbitrary messages.
+pub trait Sender<D: Clone + Codec + Send>: Clone {
```

Comment:
Is similarity to std::sync::mpsc::Sender intented?

Reply:
Internally it will usually use `mpsc::UnboundedSender` and will be used by `aleph_bft::Network`, both influenced the design.

Reply:
Ok, so I'd recommend changing the name, because even in new_network/ we have source codes using both at once.

Reply:
This is now reexported in `mod.rs` as `SenderComponent`.